src/background/service/preference.ts

```diff
@@ -243,6 +247,19 @@ class PreferenceService {
   updateWalletSavedList = (list: []) => {
     this.store.walletSavedList = list;
   };
+  getIsFirstOpen = () => {
+    if (
+      !this.store.currentVersion ||
+      version > (this.store.currentVersion || 0)
```

Comment:
here we convert version into number for compare may cause a issue: when compare between 0161 and  100, 0161 is greater but actually `1.0.0` should be greater
how about just keep the point like `0.16.1` and use package like https://www.npmjs.com/package/compare-versions for compare?

Reply:
done